src/MicrosoftTeams.schema.json

```diff
@@ -92,17 +98,20 @@
         },
         "accentColor": {
             "type": "string",
-            "description": "A color to use in conjunction with the icon. The value must be a valid HTML color code starting with '#', for example `#4464ee`."
+            "description": "A color to use in conjunction with the icon. The value must be a valid HTML color code starting with '#', for example `#4464ee`.",
+            "pattern": "^#[0-9a-fA-F]{6}$"
         },
         "galleryTabs": {
             "type": "array",
             "description": "The set of tabs that users can optionally add to Teams",
+            "maxItems": 1,
             "items": {
                 "type": "object",
```

Comment:
We should probably also add "additionalProperties": false for all objects in the schema to discourage developers from adding useless chunks to their manifest. Settings store does for all of its schemas.